src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.Telemetry.cs

```diff
@@ -25,9 +30,9 @@ internal partial class XamlCodeGeneration
 			|| Environment.GetEnvironmentVariable("JENKINS_URL").HasValue() // https://wiki.jenkins.io/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-belowJenkinsSetEnvironmentVariables
 			|| Environment.GetEnvironmentVariable("APPVEYOR").HasValue(); // https://www.appveyor.com/docs/environment-variables/
 
-		private void InitTelemetry(ProjectInstance msbProject)
+		private void InitTelemetry(GeneratorExecutionContext context)
 		{
-			var telemetryOptOut = msbProject.GetProperty("UnoPlatformTelemetryOptOut")?.EvaluatedValue ?? "";
+			var telemetryOptOut = context.GetMSBuildPropertyValue("UnoPlatformTelemetryOptOut");
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'telemetryOptOut'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6270956)